modules/optimization/test/tests/optimizationreporter/general_opt/point_loads/gold/main_seperate_multiapps_out_OptimizationReporter_0001.csv

```diff
@@ -0,0 +1,4 @@
+grad_parameter_results,parameter_results
+-1.6192745793319e-14,-904.75673506182
+-6.5314022205822e-14,77.501243204537
+-8.8620191913968e-14,506.10843135578
```

Comment:
Edit the gold file to be the exact answer and then use a tolerance,  I think they should be something like this 
```suggestion
0,-1000
0,77
0,500
```